src/pages/landing/Landing.tsx

```diff
@@ -1,11 +1,30 @@
-import React from 'react'
+import React from 'react';
 import PracticeProgramsSearch from '../../common/components/practice-programs-search/PracticeProgramsSearch';
+import ShapeWrapper from '../../common/components/shape-wrapper/ShapeWrapper';
+import Domains, { IconTypes } from './components/domains/Domains';
 
 const Landing = () => {
   return (
-    <div className='bg-yellow w-full'>
-      <PracticeProgramsSearch />
-    </div>)
-}
+    <section className="w-full">
+      <div className="bg-yellow w-full">
+        <PracticeProgramsSearch />
+      </div>
+      <ShapeWrapper>
+        {' '}
+        <Domains
+          title="Domenii"
+          domains={[
+            { name: 'Medical', icon: IconTypes.USERS },
```

Comment:
Domeniile ar trebui sa vina dintr-un fisier separat poate de nomenclatoare + textele trebuie sa vina din traduceri.

Reply:
fixed it